receiver/githubreceiver/README.md

```diff
@@ -174,6 +174,9 @@ receivers:
             secret: ${env:SECRET_STRING_VAR}
             required_headers:
                 WAF-Header: "value"
+        scrapers: # The validation expects at least a dummy scraper config
+            scraper:
+                github_org: <some value>
```

Comment:
some_value: did you mean a specific value such as the human readable string of a github org ID?

Reply:
this is for documentation purposes, so using something as real as possible such as "open-telemetry"